app/assets/javascripts/discourse/app/mixins/dismiss-topics.js

```diff
@@ -0,0 +1,31 @@
+import Mixin from "@ember/object/mixin";
+import User from "discourse/models/user";
+import showModal from "discourse/lib/show-modal";
+import I18n from "I18n";
+
+export default Mixin.create({
+  actions: {
+    resetNew() {
+      const user = User.current();
+      if (!user.new_new_view_enabled) {
+        return this.send("callResetNew");
```

Comment:
If we remove `@action` from `callResetNew` definition, I think (I'm not 100% sure) we don't have to use `send` to call anymore and we can simply do `this.callResetNew()`.